release_notes/topics/18_0_0.adoc

```diff
@@ -50,9 +50,33 @@ There are more WebAuthn improvements and fixes in addition to that.
 For more details, see link:{adminguide_link}#_user_session_limits[{adminguide_name}].
 Thanks to https://github.com/mfdewit[Mauro de Wit] for the contribution.
 
+== Quarkus: Import realms at startup
+
+The {project_name} Quarkus distribution now supports importing your realms directly at start-up. For more information, check the corresponding https://www.keycloak.org/server/importExport[guide].
+
+== Quarkus: JSON and File Logging improvements
+
+The {project_name} Quarkus distribution now initially supports logging to a File and logging structured data using JSON.
+
+For more information on the improvements, check the corresponding https://www.keycloak.org/server/logging[guide].
+
+== Quarkus: Environment variable expansion for values in keycloak.conf
+
+The {project_name} Quarkus distribution now supports expanding values in keycloak.conf from environment variables, e.g. `log-level=${ROOT_LOG_LVL:info}` would look for an environment variable names `ROOT_LOG_LVL` first and default to info if it could not find it.
+
+For more information, check the corresponding https://www.keycloak.org/server/configuration[guide].
+
+== Quarkus: New Option db-url-port
+
+You can now change the port of your jdbc connection string explicitly by setting the new `db-url-port` configuration option. As for the other convenience options, this option will be overridden by the value of a full `db-url`, if set.
+
+== Quarkus: Split metrics-enabled option into health-enabled and metrics-enabled
+The `metrics-enabled` option now only enables the metrics for {project_name}. To enable the readiness and liveness probe, there's the new build option `health-enabled`. This allows more fine-grained usage of these options.
+
 == Other improvements
 
 * Account console alignments with latest PatternFly release.
 * Support for encrypted User Info endpoint response. Thanks to https://github.com/giacomoa[Giacomo Altiero]
 * Support for the algorithm RSA-OAEP with A256GCM used for encryption keys. Thanks to https://github.com/fbrissi[Filipe Bojikian Rissi]
 * Support for login with GitHub Enterprise server. Thanks to https://github.com/nngo[Neon Ngo]
+* Multiple bugfixes and smaller improvements for the Quarkus distribution (e.g. hostname settings, start-up performance)
```

Comment:
Small improvements/bugfixes are listed in the full release notes so don't need a separate mention here

```suggestion
```